acmw-db-app/pages/execdashboard.js

```diff
@@ -49,7 +59,9 @@ const ExecDashboard = () => {
             <main className={styles.main}>
                 <SubmitButton label="Update GHC Volunteer Hours" handleChange={() => {setShowGHCForm(true)}}/>
                 {showGHCForm && <GHCVolunteerForm closeForm={() => {setShowGHCForm(false)}}/>}
-                <StudentSearch />
+                <StudentSearch selectStudent={student => selectStudent(student)}/>
```

Comment:
The student search component should actually be in the add account form component, not just joined together in the execdashboard